messenger.rst

```diff
@@ -1131,6 +1131,35 @@ during a request::
         :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`
         or :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\WebTestCase`.
 
+Amazon SQS
+~~~~~~~~~~
+
+.. versionadded:: 5.1
+
+    The Amazon SQS transport has been added in Symfony 5.1
+    Install it by running:
+
+    .. code-block:: terminal
+
+        $ composer require symfony/amazon-sqs-messenger
+
+The ``SQS`` transport configuration looks like this:
+
+.. code-block:: bash
+
+    # .env
+    MESSENGER_TRANSPORT_DSN=sqs://guest:guest@sqs.eu-west-3.amazonaws.com/test?region=eu-west-3
```

Comment:
I would also add an option to debug with local fake sqs

```
MESSENGER_TRANSPORT_DSN=sqs://localhost:9494/messages?sslmode=disable
```

Reply:
@jderusse Your example with slashes in the password won't work since it will be parsed by `parse_url`. So it would be good to add an example using the options.

```suggestion
MESSENGER_TRANSPORT_DSN=sqs://sqs.eu-west-3.amazonaws.com/23982323/queue-name
```

And add this in the example since at least a lot of AWS secrets have slashes in them:

```framework:
    messenger:
        transports:
            async:
                dsn: '%env(MESSENGER_TRANSPORT_DSN)%'
                options:
                    access_key: "%env(AWS_KEY)%"
                    secret_key: "%env(AWS_SECRET)%"
```

Reply:
thanks.

specials char in credentials works but you *have to* urlencode theme.
indeed my exemple is buggy because of this.

I'd rather keep (and fix) my exemple and add a note about that in the doc. Many people are confused about this.

Reply:
A lot of AWS keys will have slashes in them, and I also don't like you have to put your AWS credentials in the .env file multiple times since it's in the DSN in the example.

At least mention this in docs so people don't have to find this out.

Reply:
Agreed. Both ways should be documented. 

I like your sample. A little bit more verbose, but avoid missconfiguraiton.
But url_encoding credential is hard to find in the current documentation and should also be documented